src/plugins/intel_npu/tests/functional/CMakeLists.txt

```diff
@@ -56,6 +60,8 @@ if(MSVC)
   target_compile_options(${TARGET_NAME} PRIVATE /Zc:preprocessor)
 endif()
 
+target_compile_definitions(${TARGET_NAME} PRIVATE CI_BUILD_NUMBER=\"mock_version\")
```

Comment:
Why do you need those changes in CMake?

Reply:
It is taken from hetero plugin tests. I use this `CI_BUILD_NUMBER` in mock plugin's versions.